### PR TITLE
Rainbow slime extracts don't get deleted after a use if they have more than one use.

### LINKED ISSUE #45003: Slime extracts always get used, regardless of extra uses
Round ID: 112428
BYOND version: 512.1472

Admin spawned recurring rainbow slime extracts were always consumed no matter how many uses they had left when I injected them with 5u of plasma. Admin spawned them many times and this issue occurred every single time. It had the desired effect but the extract disappeared. I tried extending the uses of a normal rainbow extract created by one of the previous reactions, but it was also consumed, regardless of the extra use.

### PR DESCRIPTION
**Pull request by Firecage**
## About The Pull Request

This PR's main change is that the rainbow slime extract no longer gets deleted from its reaction with plasma and slime jelly if it still has uses left.

The other change is adding a simple logging so admins are notified when a Xenobiologist(or random tider or Captain) uses the slime jelly reaction, the volatile one, which can make things go quite bad.

Closes #45003

## Why It's Good For The Game

Well, it probably does not matter to much anymore regardless, though this would likely have been better served if it happened prior to crossbreeding, since currently we can effectively get infinite rainbow slime extracts now.

Regardless of that though, even if it was an intentional design, it is fairly silly that, unlike the other extracts, this one would suddenly end up deleted even if it had, say, 5 more uses left.

Now, from the normal reaction using plasma I see no big problems, just the scientist getting a bunch of random slime extracts to play with. A problem I do see though, would be with the volatile reaction, since there probably would be someone who would abuse the hell out of them. Ideas would be welcome.

## Changelog
:cl:
balance: Rainbow slime extracts, when they have more than one use, are no longer deleted when injected with plasma or slime jelly.
/:cl:

### CODE REVIEW

**Comment by Ty-the-Smonk** [comment]
Change balance to fix

**Comment by Firecage** [comment]
Ty, this isn't really a 'fix' though, since apparently them being deleted was intended. I might be wrong, but that is what it seems like. So this is more balance related than bug fix.

**Comment by ExcessiveUseOfCobblestone** [comment]
probably copypasta from grenades or something

**Comment by Firecage** [comment]
You know, in retrospect, they can't even abuse the extracts being multi-use anymore than they could currently abuse the single-use ones...since loading hordes of them into biobags? Yah...And then there's still the thing of getting and carrying enough slime jelly on you.

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -666,12 +665,19 @@
...
 
 /datum/chemical_reaction/slime/slimebomb/on_reaction(datum/reagents/holder, created_volume)
-	var/obj/item/grenade/clusterbuster/slime/volatile/S = new (holder.my_atom.loc)
+	var/obj/item/grenade/clusterbuster/slime/volatile/S = new (get_turf(holder.my_atom))
 	S.visible_message("<span class='danger'>Infused with slime jelly, the core begins to expand uncontrollably!</span>")
 	S.icon_state = "[S.base_state]_active"
 	S.active = TRUE
 	addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand(15,60))
-	qdel(holder.my_atom) //deleto
+	var/turf/T = get_turf(holder.my_atom)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: if you put this above 668 you don't have to call it twice

Comment (Firecage) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Alright, it is done.

**Pull request merged**